enclave_build/src/docker.rs

```diff
@@ -325,10 +346,15 @@ impl DockerUtil {
             }
 
             let act = async {
-                match self.docker.images().get(&self.docker_image).inspect().await {
+                match self.docker.inspect_image(&self.docker_image).await {
                     Ok(image) => Ok((
-                        image.config.entrypoint.unwrap(),
-                        image.config.env.ok_or_else(Vec::<String>::new).unwrap(),
+                        image.config.clone().unwrap().entrypoint.unwrap(),
```

Comment:
nit: Should have a comment why `unwrap` is safe, e.g. because it was checked before. Or even better would be to restructure the code to use the checked value from above.